README.adoc

```diff
@@ -174,6 +174,20 @@ See <<message, Message>>.
 | The label `Perso` does not exist.
 |===
 
+[[severity-level]]
+==== Severity level
+
+The severity can be one of the following:
+
+* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues that prevent them from being executed in an optimal way.
```

Comment:
I would more say that:
* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues where we believe the query most probably is wrong and/or should be updated. This includes: deprecations, experimental features, trying to match on non-existing labels etc. It should be clear what the problem is and it is also good to tell the possible cause, and the consequences, as well as give an example(s) and suggestion(s) for improvement.

Reply:
I think the most important part is that "we are (almost) certain it is a problem" and we know how to fix your query. So for:

- Deprecations -> you need to update the query with new functionallity
- Experimental features -> These can not be trusted and should be updated if  used in production. 
- Non-existing label: This query will never give any result, so it's either useless to run OR misspelled label (unless you add a node with the label, but then the notification would disappear).

Reply:
I like that, and I will add it as it is.